distributed/scheduler.py

```diff
@@ -314,124 +342,110 @@ def _to_dict(self, *, exclude: "Container[str]" = ()) -> dict:
+    address: str
 
-    .. attribute:: processing: {TaskState: cost}
+    pid: int
+    name: object
 
-       A dictionary of tasks that have been submitted to this worker.
-       Each task state is associated with the expected cost in seconds
-       of running that task, summing both the task's expected computation
-       time and the expected communication time of its result.
+    #: The number of CPU threads made available on this worker
+    nthreads: int
 
-       If a task is already executing on the worker and the excecution time is
-       twice the learned average TaskGroup duration, this will be set to twice
-       the current executing time. If the task is unknown, the default task
-       duration is used instead of the TaskGroup average.
+    #: Memory available to the worker, in bytes
+    memory_limit: int
 
-       Multiple tasks may be submitted to a worker in advance and the worker
-       will run them eventually, depending on its execution resources
-       (but see :doc:`work-stealing`).
+    local_directory: str
+    services: dict[str, int]
 
-       All the tasks here are in the "processing" state.
+    #: Output of :meth:`distributed.versions.get_versions` on the worker
+    versions: dict[str, Any]
 
-       This attribute is kept in sync with :attr:`TaskState.processing_on`.
+    #: Address of the associated :class:`~distributed.nanny.Nanny`, if present
+    nanny: str
 
-    .. attribute:: executing: {TaskState: duration}
+    #: Read-only worker status, synced one way from the remote Worker object
+    status: Status
 
-       A dictionary of tasks that are currently being run on this worker.
-       Each task state is asssociated with the duration in seconds which
-       the task has been running.
+    #: Cached hash of :attr:`~WorkerState.address`
+    _hash: int
 
-    .. attribute:: has_what: {TaskState}
+    #: The total memory size, in bytes, used by the tasks this worker holds in memory
+    #: (i.e. the tasks in this worker's :attr:`~WorkerState.has_what`).
+    nbytes: int
 
-       An insertion-sorted set-like of tasks which currently reside on this worker.
-       All the tasks here are in the "memory" state.
+    #: The total expected runtime, in seconds, of all tasks currently processing on this
+    #: worker. This is the sum of all the costs in this worker's
+    # :attr:`~WorkerState.processing` dictionary.
+    occupancy: float
 
-       This is the reverse mapping of :class:`TaskState.who_has`.
+    #: Worker memory unknown to the worker, in bytes, which has been there for more than
+    #: 30 seconds. See :class:`MemoryState`.
+    _memory_unmanaged_old: int
 
-    .. attribute:: nbytes: int
+    #: History of the last 30 seconds' worth of unmanaged memory. Used to differentiate
+    #: between "old" and "new" unmanaged memory.
+    #: Format: ``[(timestamp, bytes), (timestamp, bytes), ...]``
+    _memory_unmanaged_history: deque[tuple[float, int]]
```

Comment:
These two attributes were undocumented (and thus private before), and I felt they should remain private.